samza-yarn/src/main/java/org/apache/samza/job/yarn/FileSystemImplConfig.java

```diff
@@ -61,26 +61,15 @@ public FileSystemImplConfig(final Config config) {
   }
 
   /**
-   * Get the fs.&lt;scheme&gt;impl as the config key from scheme
+   * Get the config subset for fs.&lt;scheme&gt;.impl
+   * It can include config for fs.&lt;scheme&gt;.impl and additional config for the subKeys fs.&lt;scheme&gt;.impl.* from the configuration
+   * e.g. for scheme "myScheme", there could be config for fs.myScheme.impl, fs.myScheme.impl.client and fs.myScheme.impl.server
    * @param scheme scheme name, such as http, hdfs, myscheme
-   * @return fs.&lt;scheme&gt;impl
+   * @return a set of sub configurations without stripping off prefix
```

Comment:
nit:
`@return config for the particular scheme`

`set` of sub configurations is slightly mis-leading. Probably not worth calling out that the pre-fix is not stripped.